dev/sbt-plugin/src/sbt-test/sbt-plugin/http-backend-switch/akka-http-impl/src/main/java/com/example/hello/impl/AkkaHttpServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ */
```

Comment:
What's the empty comment block for? Should this have the copyright header in it?

Reply:
Hmm, that's weird. IT's very likely the `HeaderPlugin` is not processing these files (code under `scripted`)  but the `g8` templates don't have that either. I probably copy/pasted these originally and dragged this...